src/entities/DFGVariable.jl

```diff
@@ -78,24 +78,16 @@ mutable struct PackedVariableNodeData
 end
 
 
-abstract type AbstractVariableEstimate end
 """
     $TYPEDEF
 
 Data container to store Parameteric Point Estimate (PPE) from a variety of types.
-
-Notes
-- `ppeType` is something like `:max/:mean/:modefit` etc.
-- `solveKey` is from super-solve concept, starting with `:default`,
-- `estimate` is the actual numerical estimate value,
-- Additional information such as how the data is represented (ie softtype) is stored alongside this data container in the `DFGVariableSummary` container.
-"""
-struct VariableEstimate <: AbstractVariableEstimate
-  solverKey::Symbol
-  ppeType::Symbol
-  estimate::Vector{Float64}
+"""
+struct PointParametricEstimates
```

Comment:
`struct PointParametricEstimates <: AbstractPointParametric` as originally suggested by @Affie.

or whatever the chosen name for `AbstractPointParametric` is in the end

Reply:
from #148

`<: AbstractVariableEstimate`